src/shared/lotus-client/Lotus.ts

```diff
@@ -92,12 +94,12 @@ export class Lotus {
 
         //appStore.logsStore.logDebug(`Lotus.gesGasEstimation response:\n ${inspect(response.data)}\n`);
 
-        message.gaslimit = response.data.result.GasLimit * 2
-        message.gasfeecap = response.data.result.GasFeeCap
+        message.gaslimit = response.data.result.GasLimit * 2.4
+        message.gasfeecap = minCurrentBaseFee //response.data.result.GasFeeCap
```

Comment:
Need to use `Math.max`, but if it's a BigNumber we have to use a function that works with BigNumber